helm_lib/charts/deckhouse_lib_helm/templates/_csi_controller.tpl

```diff
@@ -172,7 +173,11 @@ spec:
       {{- include "helm_lib_priority_class" (tuple $context "system-cluster-critical") | nindent 6 }}
       {{- include "helm_lib_node_selector" (tuple $context "master") | nindent 6 }}
       {{- include "helm_lib_tolerations" (tuple $context "any-node" "with-uninitialized") | nindent 6 }}
+{{- if $context.Values.global.enabledModules | has "csi-nfs" }}
+      {{- include "helm_lib_module_pod_security_context_runtime_default" . | nindent 6 }}
+{{- else }}
```

Comment:
i think this code from another PR

Reply:
It's weird, I'll rebase.

Reply:
This code from helm lib, library has been updated and this feature has been added